### PR TITLE
Push image to Docker Hub on releases and pushes to master

### PR DESCRIPTION
**Pull request by stone-z**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Re: https://github.com/toniblyx/prowler/issues/476
This adds github actions for building and pushing the image to Docker Hub. One action for releases and one for latest.

To use: add the `DOCKER_USERNAME` and `DOCKER_PASSWORD` secrets to your GitHub repo settings

### CODE REVIEW

File: .github/workflows/docker-hub-latest.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Publish Docker
+        uses: elgohr/Publish-Docker-Github-Action@2.12
+        with:
+            name: st0nez/prowler
```
Comment (stone-z) [inline .github/workflows/docker-hub-latest.yml]:
```suggestion
            name: toniblyx/prowler
```

File: .github/workflows/docker-hub-release.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Get release version
+        id: get_version
+        run: echo ::set-env name=RELEASE_VERSION::$(echo ${GITHUB_REF:10})
+      - name: Publish Docker
+        uses: elgohr/Publish-Docker-Github-Action@2.12
+        with:
+            name: st0nez/prowler
```
Comment (stone-z) [inline .github/workflows/docker-hub-release.yml]:
```suggestion
            name: toniblyx/prowler
```

**Comment by toniblyx** [comment]
see issue  #476

**Comment by stone-z** [comment]
Closing in favor of using Docker Hub builds

**Comment by toniblyx** [comment]
Hey @stone-z look at this `docker pull toniblyx:2.2.0`, new version with new tag as you asked. I'll do that from now on each new release. Thanks your your suggestion!

**Comment by stone-z** [comment]
Awesome, this is a big help - thanks again for all your work here!

**Pull request closed without merging**